collect_app/src/main/java/org/odk/collect/android/formlists/blankformlist/BlankFormListActivity.kt

```diff
@@ -75,16 +74,7 @@ class BlankFormListActivity : LocalizedActivity(), OnFormItemClickListener {
             setResult(RESULT_OK, Intent().setData(formUri))
         } else {
             // caller wants to view/edit a form, so launch formentryactivity
-            Intent(this, FormEntryActivity::class.java).apply {
-                action = Intent.ACTION_EDIT
-                data = formUri
-                putExtra(
```

Comment:
Was this redundant? You don't have this in `FormNavigator.newInstanceIntent(this, formUri)`

Reply:
Ah good catch! I forgot to come back to this, as far as our tests go, the extras here are redundant. I'll investigate what's going on.

Reply:
Right it didn't even make sense here in the list of blank forms to use `EDIT_SAVED`.